src/content/learn/scaling-up-with-reducer-and-context.md

```diff
@@ -229,30 +229,30 @@ And `TaskList` passes the event handlers to `Task`:
 />
 ```
 
-In a small example like this, this works well, but if you have tens or hundreds of components in the middle, passing down all state and functions can be quite frustrating!
+Dalam contoh kecil seperti ini, cara ini dapat berfungsi dengan baik, namun jika Anda memiliki puluhan atau ratusan komponen di tengah, meneruskan semua *state* dan fungsi dapat sangat menjengkelkan!
 
-This is why, as an alternative to passing them through props, you might want to put both the `tasks` state and the `dispatch` function [into context.](/learn/passing-data-deeply-with-context) **This way, any component below `TaskApp` in the tree can read the tasks and dispatch actions without the repetitive "prop drilling".**
+Inilah mengapa, sebagai alternatif untuk melewatkan melalui props, Anda mungkin ingin menempatkan baik *state* `tugas` maupun fungsi `dispatch` [ke dalam _context_](/learn/passing-data-deeply-with-context) . **Dengan cara ini, komponen apa pun di bawah `TaskApp` dalam pohon(*tree*) dapat membaca tugas dan melakukan aksi *dispatch* tanpa “*prop drilling*” yang berulang.**
 
-Here is how you can combine a reducer with context:
+Berikut adalah cara menggabungkan *reducer* dengan conteks:
 
-1. **Create** the context.
-2. **Put** state and dispatch into context.
-3. **Use** context anywhere in the tree.
+1. **Buatlah** *context*.
+2. **Letakkan** *state* dan *dispatch* ke dalam *context*.
+3. **Gunakan** *context* di mana saja dalam *tree*.
 
-### Step 1: Create the context {/*step-1-create-the-context*/}
+### Langkah 1: Buat conteks {/*step-1-create-the-context*/}
 
-The `useReducer` Hook returns the current `tasks` and the `dispatch` function that lets you update them:
+Hook `useReducer` mengembalikan `tugas` saat ini dan fungsi `dispatch` yang memungkinkan Anda memperbarui tugas:
```

Comment:
```suggestion
Hook `useReducer` mengembalikan `tasks` saat ini dan fungsi `dispatch` yang memungkinkan Anda memperbaruinya:
```

We shouldn't translate `tasks` because it refers to the words being used in the code examples.